pkg/helm/actions/install_chart_test.go

```diff
@@ -308,3 +312,94 @@ func TestInstallChartBasicAuth(t *testing.T) {
 		})
 	}
 }
+
+func TestInstallChartAsync(t *testing.T) {
+	tests := []struct {
+		releaseName  string
+		chartPath    string
+		createSecret bool
+		chartName    string
+		chartVersion string
+		indexEntry   string
+		namespace    string
+		requireError bool
+		helmCRS      []*unstructured.Unstructured
+	}{
+		{
+			releaseName:  "myrelease",
+			chartPath:    "http://localhost:9181/charts/influxdb-3.0.2.tgz",
+			chartName:    "influxdb",
+			chartVersion: "3.0.2",
+			createSecret: true,
+			namespace:    "test",
+			indexEntry:   "influxdb--without-tls",
+			helmCRS: []*unstructured.Unstructured{
+				{
+					Object: map[string]interface{}{
+						"apiVersion": "helm.openshift.io/v1beta1",
+						"kind":       "HelmChartRepository",
+						"metadata": map[string]interface{}{
+							"name": "without-tls",
+						},
+						"spec": map[string]interface{}{
+							"connectionConfig": map[string]interface{}{
+								"url": "http://localhost:9181",
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.releaseName, func(t *testing.T) {
+			store := storage.Init(driver.NewMemory())
+			actionConfig := &action.Configuration{
+				RESTClientGetter: FakeConfig{},
+				Releases:         store,
+				KubeClient:       &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+				Capabilities:     chartutil.DefaultCapabilities,
+				Log:              func(format string, v ...interface{}) {},
+			}
+			objs := []runtime.Object{}
+			client := K8sDynamicClientFromCRs(tt.helmCRS...)
+			clientInterface := k8sfake.NewSimpleClientset(objs...)
+			coreClient := clientInterface.CoreV1()
+			var rel *Secret
+			var err error
+			go func() {
+				rel, err = InstallChartAsync(tt.namespace, tt.releaseName, tt.chartPath, nil, actionConfig, client, coreClient, true, tt.indexEntry)
```

Comment:
All other tests uses `fileCleanUp = false`, does this make sense here as well?

```suggestion
				rel, err = InstallChartAsync(tt.namespace, tt.releaseName, tt.chartPath, nil, actionConfig, client, coreClient, false, tt.indexEntry)
```

Reply:
Will not make a difference here but yeah good to be consistent